src/Modules/Projects/Components/ProjectList/ProjectList.tsx

```diff
@@ -1,7 +1,70 @@
-import React from 'react'
+import { useEffect, useState } from 'react'
+import { AuthorizedToken, PROJECT_URLS } from '../../../../constans/END_POINTS';
+import axios from 'axios';
+import { format } from 'date-fns';
+import NoData from '../../../Shared/Components/NoData/NoData';
+import { Link, useNavigate } from 'react-router-dom';
+
+
 
 export default function ProjectList() {
+
+  
+  let [projectsList, setProjectsList] =  useState([]);
+  let getCategoriesList = async ()=>{
```

Comment:
We need to update the name of the function.
Also if we installed the eslint extention,  we will get a warning that we are using let with something that we are not changing.

Reply:
the function name was updated,
but it had no problem because it was called in the useEffect with the same name,
eslint extention is already installed ,but we didn't recieve any warning

Reply:
are you getting any eslint warnning?
as I downloaded the code and this rule is part of the default eslint installed with vite, so it should be erroring.
![image](https://github.com/user-attachments/assets/4ec177f6-12c4-479c-be96-6175fc234c0e)

This is the extention that I mean:
![image](https://github.com/user-attachments/assets/ce4882d7-5837-4159-a174-aacc0eb6b402)

Reply:
![47dff844-a591-4f21-8e98-b2a0ca9f59d7](https://github.com/user-attachments/assets/53b500b6-bab8-4e18-98e6-1d80937aa247)
 ok, got it